config/swagger.yaml

```diff
@@ -247,6 +257,43 @@ paths:
           schema:
             $ref: '#/definitions/Error'
       security: []
+
+## CAUTION: debug endpoints, implementation may be inefficient
+  /balances:
+    get:
+      tags:
+        - external
```

Comment:
Comment suggests internal, this point mentions external. It looks to me it shall be internal.

Reply:
The thing is that it prints the whole state tree, which may be an easy point for DDoS attack, hence the warning.

External will make debugging of UAT/testnet envs easier (you may e.g. check genesis/state trees on all UAT/testnet nodes). Internal makes it available only from your own running node (or from UAT/testnet when you log into the node).